src/cpp/main.cpp

```diff
@@ -55,6 +55,37 @@ namespace jank
     //);
   }
 
+  void run_main(util::cli::options const &opts, runtime::context &rt_ctx)
+  {
+    {
+      profile::timer timer{ "require clojure.core" };
+      rt_ctx.load_module("/clojure.core").expect_ok();
+    }
+
+    {
+      profile::timer timer{ "eval user code" };
+      rt_ctx.load_module("/" + opts.target_module).expect_ok();
+
+      auto main_var = rt_ctx.find_var(opts.target_module, "-main").unwrap_or(nullptr);
```

Comment:
I changed this to be:

```cpp
auto const main_var(rt_ctx.find_var(opts.target_module, "-main").unwrap_or(nullptr));
```

1. We always use direct initialization
2. Whatever can be const must be